examples/cluster-workflow-template/workflow-template-ref.yaml

```diff
@@ -2,6 +2,8 @@ apiVersion: argoproj.io/v1alpha1
 kind: Workflow
 metadata:
   generateName: cluster-workflow-template-hello-world-
+  labels:
+    workflows.argoproj.io/no-test-environment: "true"
```

Comment:
If you set this to false, the current code will not correctly handle it

Reply:
We fixed that by reverting to a single label called `no-test` and using the value of such label to express why this example can't be tested. In this way there is no really a point to support "true" because "no-test" is there only if you want to no-test it